src/constants.js

```diff
@@ -1,13 +1,13 @@
 import { io } from 'socket.io-client'
 
-export const userFetchInterval = 3 * 1000
-export const heartbeatInterval = 15 * 1000
-export const myGameFetchInterval = 3 * 1000
-export const gameStatus = {
+export const USER_FETCH_INTERVAL = 3 * 1000
+export const HEARTBEAT_INTERVAL = 15 * 1000
+export const GAME_FETCH_INTERVAL = 3 * 1000
+export const GAME_STATUS = {
   STARTED: 'STARTED',
   FINISHED: 'FINISHED',
 }
 export const JOIN_ROOM = 'join_room'
 export const GAME_STATE = 'game_state'
 export const MOVE_PLAYED = 'move_played'
```

Comment:
As stated [here](https://github.com/tic-tac-toe-devs/tic-tac-toe-backend/pull/24#discussion_r988619211):
In the protocol description we agreed on "Event types: GAME_STATE , MOVE_PLAYED" , so the property values should also be upper cased, along with the property names.
That being the case, we might as well change join_room as well. This also needs to be communicated with the be